src/options.test.ts

```diff
@@ -14,11 +16,14 @@ describe('when attempting to get the configuration for codehawk', () => {
         good: 'yes',
       }
 
-      require('fs').__codehawkConfigFound(true, mockCodehawkJson)
+      ;(fs.existsSync as jest.Mock).mockReturnValue(true)
```

Comment:
semicolons wat?

Reply:
This is an automated prettier fix :(

I guess the interpreter looks at the object above and attempts to "call" it with the `fs.existsSync`

Reply:
kinda crazy! but fair enough

Reply:
Fixed!